pyproject.toml

```diff
@@ -53,10 +53,12 @@ lint.select = [
     "W",  # PYCODESTYLE WARNINGS
 ]
 
+
```

Comment:
?

Reply:
ruff format wasn't fixing my lint errors, was trying to debug (did not succeed and fixed the lines manually), will remove